tests/pytests/unit/runners/test_network.py

```diff
@@ -0,0 +1,255 @@
+            "fxsr",
+            "sse",
+            "sse2",
+            "ht",
+            "syscall",
+            "nx",
+            "rdtscp",
+            "lm",
+            "constant_tsc",
+            "rep_good",
+            "nopl",
+            "xtopology",
+            "nonstop_tsc",
+            "eagerfpu",
+            "pni",
+            "pclmulqdq",
+            "monitor",
+            "ssse3",
+            "cx16",
+            "pcid",
+            "sse4_1",
+            "sse4_2",
+            "x2apic",
+            "movbe",
+            "popcnt",
+            "aes",
+            "xsave",
+            "avx",
+            "rdrand",
+            "hypervisor",
+            "lahf_lm",
+            "abm",
+            "3dnowprefetch",
+            "invpcid_single",
+            "fsgsbase",
+            "avx2",
+            "inv    pcid",
+            "rdseed",
+            "clflushopt",
+            "md_clear",
+            "flush_l1d",
+        ],
+        "os_family": "RedHat",
+        "osarch": "x86_64",
+        "mem_total": 1998,
+        "swap_total": 2047,
+        "biosversion": "VirtualBox",
+        "productname": "VirtualBox",
+        "manufacturer": "innotek GmbH",
+        "biosreleasedate": "12/01/2006",
+        "uuid": "dd95fedd-1a2b-5e48-86a7-7e339f9f02a1",
+        "serialnumber": "0",
+        "virtual": "VirtualBox",
+        "ps": "ps -efHww",
+        "osrelease_info": [7, 8, 2003],
+        "osmajorrelease": 7,
+        "osfinger": "CentOS Linux-7",
+        "path": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin",
+        "systempath": ["/usr/local/sbin", "/usr/local/bin", "/usr/sbin", "/usr/bin"],
+        "pythonexecutable": "/usr/bin/python3",
+        "pythonpath": [
+            "/usr/bin",
+            "/usr/lib64/python36.zip",
+            "/usr/lib64/python3.6",
+            "/usr/lib64/python3.6/lib-dynload",
+            "/usr/lib64/python3.6/site-packages",
+            "/usr/lib/python3.6/site-packages",
+        ],
+        "pythonversion": [3, 6, 8, "final", 0],
+        "saltpath": "/usr/lib/python3.6/site-packages/salt",
+        "saltversion": "3003",
+        "saltversioninfo": [3003],
+        "zmqversion": "4.1.4",
+        "disks": ["sda", "sr0"],
+        "ssds": [],
+        "shell": "/bin/sh",
+        "lvm": {"centos": ["root", "swap"]},
+        "mdadm": [],
+        "username": "root",
+        "groupname": "root",
+        "pid": 2469,
+        "gid": 0,
+        "uid": 0,
+        "zfs_support": False,
+        "zfs_feature_flags": False,
+    }
+}
+
+
+@pytest.fixture
+def configure_loader_modules():
+
+    return {
+        network: {
+            "__grains__": {
+                "osarch": "x86_64",
+                "os_family": "Redhat",
+                "osmajorrelease": 7,
+                "kernelrelease": "3.10.0-1127.18.2.el7.x86_64",
+            },
+        },
+    }
+
+
+def test_wolmatch():
+    """
+    Test wolmatch
+    """
+    MINION_ID = "test-host"
```

Comment:
looks like you are setting this variable here in the function and at a global function. Can one be removed?

Reply:
Yeah, was originally going to make a fuller test of the functionality unrelated to my fix, and then thought why do that, just write the test for my fix, have too much else to do and left some of that stucrture around so that it could be extended later, for whenever further tests are around

But on reflection, fixtures, don't agree with forcing everyone to use them as ordained by Salt.
If a contributor writes a pytest their way and it works, it should be acceptable, rather than having to rely on Salt frameworks, so long as their test is reasonable.

Understand as a core member I have to use fixtures, but don't think we should enforce that on contributors